package.json

```diff
@@ -100,9 +101,9 @@
 		"format:check": "lerna run format:check",
 		"format": "lerna run format",
 		"gen-util": "yo ./scaffolder/index.js",
-		"lint": "eslint --quiet --ext .ts ./packages/**/src",
-		"lint:warnings": "eslint --ext .ts ./packages/**/src",
-		"lint:fix": "eslint --quiet --ext .ts --fix ./packages/**/src",
+		"lint": "eslint --quiet ./packages/**/src/**/*.ts",
```

Comment:
Might be good to verify these changes don't re-introduce the issue where not all files were being linted. I think you can verify this by temporarily removing [this line](https://github.com/Esri/hub.js/pull/2100/files#diff-a32a0887ed9d1d707bbb3b845b7df7fd40e673c47e7b60a3ebd896b68d3b8839R105) and running this lint command. If it properly flags the enums in the orval-generated dirs, then these scripts should be working correctly.

Reply:
fixed in 6b47a958c4379dea7cc3b53635179be0c817eba5